.travis.yml

```diff
@@ -32,14 +32,10 @@ before_script:
   - sleep 3 # wait for xvfb to boot
   - yarn build-all
 script:
-  - yarn test-cli-formatting
-  - yarn test-launcher-formatting
-  - yarn lint
-  - yarn unit
-  - yarn closure
+  - yarn test-cli-formatting & yarn test-launcher-formatting & yarn lint & yarn unit & yarn closure & yarn compile-devtools & wait
```

Comment:
how much of the time difference is a result of this change does it make reading the output a lot more difficult?